src/layers.rs

```diff
@@ -281,6 +295,64 @@ impl LayersManager {
         }
     }
 
+    pub fn turn_alias_on(&mut self, name: String) {
+        if let Some(index) = self.get_idx_from_alias(name) {
+            let idx = index.clone();
+            self.turn_index_on(idx);
+        }
+    }
+    pub fn turn_alias_off(&mut self, name: String) {
+        if let Some(index) = self.get_idx_from_alias(name) {
+            let idx = index.clone();
+            self.turn_index_off(idx);
+        }
+    }
+
+    // used as short hand to check state prior to turning off. Essentially toggle, but explicit
+    pub fn turn_index_off(&mut self, idx: usize) {
```

Comment:
I think we can safely adjust `turn_layer_off/on` to do this by default.

I think that these two new functions introduce a bit of ambiguity.
I.E if you only had access to the function names. What would you expect
`turn_layer_on` to do vs `turn_index_on`

Reply:
Yep agree - almost did that but figured I wouldn't touch it for now and could adjust later. Will change